onedal/covariance/covariance.py

```diff
@@ -0,0 +1,62 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import numpy as np
+from sklearn.utils import check_array
+
+from daal4py.sklearn._utils import get_dtype, make2d
+from onedal import _backend
+
+from ..common._policy import _get_policy
+from ..datatypes import _convert_to_supported, from_table, to_table
+
+
+class BaseCovariance:
+    def __init__(self):
+        pass
```

Comment:
```suggestion
    def __init__(self, method):
        self.method= method
```

Reply:
I think these changes are redundant since we only have one available method for Covariance at this moment.

Reply:
We have this on our OneDAL API, it worth have it for onedal4py and sklearnex ifaces as well.
For some of our OneDAL estimators we sometimes have params with one able setting, so for consistency it worth have here as well.